### DISCUSSION TITLE
Restart after changing the shape of the EB

### DISCUSSION THREAD

**Discussion by ZhaofanZhu** [AMReX-Combustion/PeleLMeX#239] [Q&A] [answered]
Hello,
I am trying a case with an EB. I want to change the shape of the EB (specifically, change the wavelength of a sinusoidal plane) after calculating for a period of time, and then restart the calculation. 
I tried using the "amr.restart", but it failed with "amrex::Abort::0::MLMG failing so lets stop here !!!".
The "ht.pltfile" in the documentation also tried, didn't work. (The program seems to ignore this parameter)
Is there any way to achieve this process?
Looking forward to any reply, thanks in advance!

**Comment by baperry2** [comment, marked as answer]
It is currently not supported to restart with a changed EB geometry. There's a fundamental issue here: if new cells become uncovered, there would be no consistent way to populate them with state data.

**Reply by ZhaofanZhu** [reply]
Okay, I understand, thank you.
